### DISCUSSION TITLE
Why does Mineflayer depend on itself?

### DISCUSSION THREAD

**Discussion by interrrp** [PrismarineJS/mineflayer#3343] [Q&A] [answered]
https://github.com/PrismarineJS/mineflayer/blob/195b3cbd70a110080af9b77a4659991c5d9e484a/package.json#L46

This causes issues when running on Bun.

**Comment by rom1504** [comment, marked as answer]
This is a dev dependency and is used for examples to work when cloning the repo

Is there an open issue in bun to fix this bug yet?

**Reply by interrrp** [reply]
It now works on Bun 1.1.3. Thanks!
